File: Access Modifiers.cs
using System;

class Example {
    public string PublicVar = "Public";
    private string PrivateVar = "Private";
    protected string ProtectedVar = "Protected";
    internal string InternalVar = "Internal";

    public void Show() {
        Console.WriteLine(PublicVar);
        Console.WriteLine(PrivateVar);
        Console.WriteLine(ProtectedVar);
        Console.WriteLine(InternalVar);
    }
}

class Program {
    static void Main() {
        Example obj = new Example();
        obj.Show();
        Console.WriteLine(obj.PublicVar);
        // Console.WriteLine(obj.PrivateVar); // Error: Inaccessible
    }
}

File: Access Modifiers.php
<?php
class Example {
    public $publicVar = "Public";
    private $privateVar = "Private";
    protected $protectedVar = "Protected";

    public function show() {
        echo $this->publicVar . "\n";
        echo $this->privateVar . "\n";
        echo $this->protectedVar . "\n";
    }
}

$obj = new Example();
$obj->show();

?>
